products/firefox_android/localize_android.md

```diff
@@ -0,0 +1,45 @@
+# Firefox for Android - How to localize
+
+Please make sure to go through the [Firefox for Android README](/products/firefox_android/README.md) first to get a clear overview of what this project covers.
+
+The release cycle for Firefox for Android is the same as Firefox for Desktop, which is generally 6 weeks - but can sometimes be 8 weeks. "Merge day" is when localizations move to the next train - and the last day available to include localization updates to the next Beta version if you work on tools like Pootle.
+
+You can see the [detailed schedule for each release here](https://wiki.mozilla.org/RapidRelease/Calendar).
+
+Note: please make sure that you've already localized mozilla.org before you start working on Firefox for Android or any other projects.
+
+## The importance of regularly checking your dashboard
+
+Please keep in mind during your entire localization process to regularly check [your locale's dashboard](https://l10n.mozilla.org/shipping/dashboard): this is extremely important since if you break something, and your locale is shipping on multi-locale builds (so, the builds that actually ship on the Play Store), you will need to fix it as soon as possible since it breaks the build for everyone!
```

Comment:
Do you mean I should change the link to the dashboard to the teams page? Or add the teams page (and the fact they can bookmark their own) as additional information?